### PR TITLE
Added support for mifare classic value block operations

### PR DESCRIPTION
**Pull request by taichunmin**
Added support for mifare classic value block operations

### CODE REVIEW

**Comment by taichunmin** [comment]
I didn't know how to implement send bytes to tag and then wait for response nack or timeout (which means success) in mf value operation.

截圖 2023-10-25 上午9 08 00

File: software/script/chameleon_cmd.py [hunk 1]
```diff
@@ -277,6 +277,29 @@ class CStruct(ctypes.BigEndianStructure):
...
+        :param operator:
+        :param operand:
+        :param dst_block:
+        :param dst_type:
+        :param dst_key:
+        :return:
+        """
+        data = struct.pack('!BB6sBiBB6s', src_type, src_block, src_key, operator, operand, dst_type, dst_block, dst_key)
+        resp = self.device.send_cmd_sync(Command.MF1_MANIPULATE_VALUE_BLOCK, data)
+        resp.data = resp.status == Status.HF_TAG_OK
```
Comment (doegox) [inline software/script/chameleon_cmd.py:300]: beware now it's resp.parsed = ...

File: docker-compose.yml
Comment (doegox) [inline docker-compose.yml:1]: unrelated, please use another PR if you want to add it and explain why
please remove from this PR

Comment (taichunmin) [reply docker-compose.yml:1]: OK, I will remove the file before merge

File: software/script/chameleon_cmd.py (hunk 1 above)
Comment (taichunmin) [reply software/script/chameleon_cmd.py]: Fixed

**Comment by doegox** [comment]
I didn't test specifically the new value block operations but I did some tests to make sure existing 14a support for raw, mfu etc was still working fine, so good enough for a merge ;)

**Pull request merged**